### PR TITLE
New tax exempt request doc

### PR DESCRIPTION
**Pull request by EdwardAngert**
**Note:** Please fill out the PR Template to ensure proper processing.

Closes: #5595 

## Summary

**[Request Tax Exempt Status](https://pantheon.io/docs/tax-exempt-status)**: New doc to provide customers with the materials we'd need to confirm tax exemption. [PR 5598](https://github.com/pantheon-systems/documentation/pulls/5598).

## Effect

The following changes are already committed:
- new doc with necessary tax exemption materials

## Remaining Work

The following changes still need to be completed:

- [x] Confirmation from @NeoMeteor 
- [x] add to categories
- [x] Copy Edit
- [ ] New doc! do the Post Launch things!

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5598 | Difference |
| --- | --- | --- | --- |
| Performance Score | `55.00000000000001` | `47` | `-8` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5598` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19270/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19270/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by NeoMeteor** [comment]
The documentation is good. This will likely be pretty helpful!

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5598 | Difference |
| --- | --- | --- | --- |
| Performance Score | `49` | `42` | `-7` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5598` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19341/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19341/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: source/content/tax-exempt-status.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+tags: [billing]
+categories: [manage]
+contributors: [edwardangert]
+searchboost: 150
+reviewed: "2020-03-24"
+---
+
+To make it easy for our team to confirm your tax-exempt status and have it reflected on your bill quickly, [contact Support](/support) with the following information:
+
+- Copy of tax exemption determination letter
```
Comment (alexfornuto) [inline source/content/tax-exempt-status.md]: Does this mean they need to have a copy handy, or they need to send a copy to Support? I'm guessing the latter, and that this should be clarified to say so.

Also, does it need to be in PDF format? Does a cell phone picture of the letter suffice? You know someone's gonna try that.

File: source/content/tax-exempt-status.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- Email address of Site Owner
```
Comment (alexfornuto) [inline source/content/tax-exempt-status.md]: Does this mean you need to provide the email address of the site owner (i.e. know it), or be contacting support _from_ a user associated with that email?

File: source/content/tax-exempt-status.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+- Dashboard URL of affected sites
```
Comment (alexfornuto) [inline source/content/tax-exempt-status.md]: Opportunity to [crosslink](https://pantheon.io/docs/sites)?

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5598 | Difference |
| --- | --- | --- | --- |
| Performance Score | `54` | `42` | `-12` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5598` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19397/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19397/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: source/content/tax-exempt-status.md (hunk 2 above)
Comment (EdwardAngert) [reply source/content/tax-exempt-status.md]: attempted to clarify in L11

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5598 | Difference |
| --- | --- | --- | --- |
| Performance Score | `49` | `41` | `-8` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5598` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19404/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19404/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5598 | Difference |
| --- | --- | --- | --- |
| Performance Score | `56.00000000000001` | `50` | `-6` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5598` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19405/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19405/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5598 | Difference |
| --- | --- | --- | --- |
| Performance Score | `71` | `35` | `-36` |
| Best Practice Score | `93` | `86` | `-7` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `96` | `-2` |

Lighthouse Audit Reports:

- [`pr-5598` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19599/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19599/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5598 | Difference |
| --- | --- | --- | --- |
| Performance Score | `57.99999999999999` | `49` | `-9` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5598` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19600/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19600/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by EdwardAngert** [comment]
rebased and forcepushed to see if Lighthouse thinks better of it

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5598 | Difference |
| --- | --- | --- | --- |
| Performance Score | `49` | `51` | `2` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5598` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19608/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19608/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Pull request merged**